File: nagasena/src/org/openexi/schema/HeaderOptionsSchema.java
package org.openexi.schema;

import java.net.URL;

public final class HeaderOptionsSchema {
  
  private static final String COMPILED_SCHEMA_LOCATION = "HeaderOptions.xsc";
  
  private static final EXISchema m_schema;
  
  static {
    URL optionslSchemaURI =
      HeaderOptionsSchema.class.getResource(COMPILED_SCHEMA_LOCATION);
    EXISchema schema = null;
    try {
      schema = CommonSchema.loadCompiledSchema(optionslSchemaURI);
    }
    finally {
      m_schema = schema;
    }
  }
  
  public static EXISchema getEXISchema() {
    return m_schema;
  }
  
  private HeaderOptionsSchema() {
  }

}

File: nagasenacs/src/Nagasena/Schema/HeaderOptionsSchema.cs
namespace Nagasena.Schema {

  /// <exclude/>
  public sealed class HeaderOptionsSchema {

    private const string COMPILED_SCHEMA_LOCATION = "HeaderOptions.xsc";

    private static readonly EXISchema m_schema = null;

    static HeaderOptionsSchema() {
      EXISchema schema = null;
      try {
        schema = CommonSchema.loadCompiledSchema(COMPILED_SCHEMA_LOCATION);
      }
      finally {
        m_schema = schema;
      }
    }

    public static EXISchema EXISchema {
      get {
        return m_schema;
      }
    }

    private HeaderOptionsSchema() {
    }

  }

}